app/views/settings/email/_form.html.erb

```diff
@@ -40,7 +40,7 @@
       </h3>
     </legend>
 
-    <a href="#" class="govuk-link">See sample PDF</a>
+    <%= link_to 'See sample pdf',asset_url('/sample.pdf'), class: 'govuk-link', target: '_blank' %>
```

Comment:
Just to match the prototype, can we change to 'See sample PDF'?

Reply:
Ooops! Done!